{{cookiecutter.project_name}}/noxfile.py

```diff
@@ -11,7 +11,14 @@
 
 package = "{{cookiecutter.package_name}}"
 python_versions = ["3.8", "3.7", "3.6"]
-nox.options.sessions = "pre-commit", "safety", "mypy", "tests", "typeguard"
+nox.options.sessions = (
+    "docs-build",
+    "mypy",
+    "pre-commit",
+    "safety",
+    "tests",
+    "typeguard",
+)
```

Comment:
Let's keep the existing order, with `docs-build` added at the end. This is the order in which the sessions are run, so building the documentation in the end makes sense. Also it's useful to run static checks before the test suite.

```suggestion
nox.options.sessions = (
    "pre-commit",
    "safety",
    "mypy",
    "tests",
    "typeguard",
    "docs-build",
)
```

Reply:
Done.